stistools/add_s_region.py

```diff
@@ -0,0 +1,403 @@
+#! /usr/bin/env python
+
+import glob
+import math
+import os
+import sys
+import argparse
+
+from astropy.io import fits
+from astropy.wcs import WCS
+import numpy as np
+import logging
+
+import pysiaf
+
+__version__ = '0.1'
```

Comment:
Should we be versioning based on the `stistools` package as a whole instead?

Reply:
Yes, although it means importing stistools and getting those spurious deprecation messages...